README.md

```diff
@@ -0,0 +1,33 @@
+# CAMA-CORE
+The CORE of the CAMA application is build using React-Native and the EXPO
+
+# Install
+1. Clone Repository [HowTo GIT](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)  
+    `git clone https://github.com/healthhackersER/CAMA-CORE-Example.git`
+2. Run npm installer  
+   `npm install`
+1. Run Application  
+   `npm start` or `npm run web`
+
+## Pre-requisities
+- **Node.js**: In order to install Expo CLI you will need to have Node.js (we recommend the latest stable version- but the maintenance and active LTS releases will also work) installed on your computer. [Download the recommended version of Node.js.](https://nodejs.org/en/)
+- **Git**: Additionally, you'll need Git to create new projects. [You can download Git from here.](https://git-scm.com/)
+
+# Getting Started
+1. [**React Native**](https://facebook.github.io/react-native/)
+   > _React Native_ is like _React_, but it uses native components instead of web components as building blocks. 
+So to understand the basic structure of a React Native app, you need to understand some of the basic React concepts, like JSX, components, ``state``, and ``props``.   
```

Comment:
Please describe what JSX is or put a link

Reply:
Done 😀😀